content/slice/language-guide/enum-types.md

```diff
@@ -114,12 +175,28 @@ unchecked enum MyInt16 : int16 {}
 ```
 {% /slice2 %}
 
+{% slice2 %}
```

Comment:
```suggestion

{% slice2 %}
```

I think having a white space above and below headings is always preferred.

Reply:
Looks like you inserted this blank line in the wrong spot.